web/src/components/Storyblok/StoryblokImage/StoryblokImage.tsx

```diff
@@ -45,6 +45,34 @@ export const StoryblokImage = forwardRef<HTMLImageElement, StoryblokImageProps>(
 		const webPSrc = constructStoryblokImageSrc(src, serviceOptions, "webp");
 		const avifSrc = constructStoryblokImageSrc(src, serviceOptions, "avif");
 		const jpgSrc = constructStoryblokImageSrc(src, serviceOptions, "jpeg");
+
+		// Set height & width to values provided from image service with fallback as square image at ~full size mobile resolution
+		// Will need updating if shape of image url changes; assumes https://a.storyblok.com/f/292509/648x349/be49eaa335/image-name.JPG/m/filters:format%28avif%29:quality%2880%29
+		const widthSubstr = src.match(
+			/^https:\/\/a\.storyblok\.com\/f\/\d+\/\d{2,5}x\d{2,5}/gm
+		)
+			? src.split("/")[5].split("x")[0]
+			: "";
+		const heightSubstr = src.match(
+			/^https:\/\/a\.storyblok\.com\/f\/\d+\/\d{2,5}x\d{2,5}/gm
+		)
+			? src.split("/")[5].split("x")[1]
+			: "";
+		const dimensions = {
+			width: !isNaN(parseFloat(widthSubstr)) ? widthSubstr : 600,
+			height: !isNaN(parseFloat(heightSubstr)) ? heightSubstr : 600,
+		};
+
+		if (
+			isNaN(parseFloat(widthSubstr)) ||
+			isNaN(parseFloat(heightSubstr)) ||
+			dimensions.width === 0 ||
+			dimensions.height === 0
+		) {
+			console.warn("Dimensions are not valid numbers");
```

Comment:
Change to using logger

Reply:
Decided to leave this as-is following precedent on L41 - isn't a code error so don't feel adding to logger would be right thing to do